release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -772,6 +772,8 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-12-bare-metal-hardware-bug-fixes"]
 ==== Bare Metal Hardware Provisioning
 
+* Before this update, the Ironic provisioning service did not support Baseboard Management Controllers (BMCs) that use weak eTags combined with strict eTag validation. If the BMC provides a weak eTag, Ironic returns two eTags: the original eTag and the original eTag converted to the strong format for compatibility with BMCs that do not support weak eTags. Although Ironic can send two eTags, BMCs using strict eTag validation reject such requests due to the presence of the second eTag. As a result, on some older server hardware, bare-metal provisioning failed with the following error: `HTTP 412 Precondition Failed`. In {product-title} 4.12 and later, this behavior changes and Ironic no longer attempts sending two eTags in cases where a weak eTag is provided. Instead, if a Redfish request dependent on an eTag fails with an eTag validation error, Ironic retries the request with known workarounds. This minimizes the risk of bare-metal provisioning failures on machines with strict eTag validation. (link:https://issues.redhat.com/browse/OCPBUGS-3479[*OCPBUGS#3479*])
```

Comment:
_If the BMC provides a weak eTag_ I read this as part of the bug, rather than proper behavior (right?). 
Maybe something like:
_By design, if the BMC provides a weak eTag..._

s/sending/to send ?

Do we know what a Redfish request is?

Reply:
Implemented your feedback. No action on the the Redfish bit though; I think in the context of this Bare Metal Hardware Provisioning section, this should be well understood... Redfish is a communication protocol for servers.